docs/core/runtime-config/runtimeconfig.md

```diff
@@ -0,0 +1,49 @@
+
+## File format
+
+The *runtimeconfig.json* file has the following format:
+
+```json
+{
+   "runtimeOptions": {
+      "configProperties": {
+         "config-property-name1": config-value1,
+         "config-property-name2": config-value2
+      }
+   }
+}
+```
+
+## Individual configuration settings
+
+The *runtimeconfig.json* file supports the following configuration settings:
+
+|Name|Value|Description|
+|--|--|--|
+|"Switch.System.Globalization.EnforceJapaneseEraYearRange"|true &#124; false|Determines whether range checks for calendars that support multiple eras are relaxed (`false`, the default value) or whether dates that overflow an era's date range throw an <xref:System.ArgumentOutOfRangeException> (`true`). For more information, see [Calendars, eras, and date ranges: Relaxed range checks](../../standard/datetime/working-with-calendars.md#calendars-eras-and-date-ranges-relaxed-range-checks).|
+|"Switch.System.Globalization.EnforceLegacyJapaneseDateParsing"|true &#124; false|Determines whether a string that contains either "1" or "Gannen" as the year parses successfully (`false`, the default), or whether only "1" is supported (`true`). For more information, see [Representing dates in calendars with multiple eras](../../standard/datetime/working-with-calendars.md#representing-dates-in-calendars-with-multiple-eras).|
+|"Switch.System.Globalization.FormatJapaneseFirstYearAsANumber"|true &#124; false|Determines whether the first year of a Japanese calendar era is formatted as "gannen" (`false`, the default) or as a number. For more information, see [Representing dates in calendars with multiple eras](../../standard/datetime/working-with-calendars.md#representing-dates-in-calendars-with-multiple-eras).|
+|"System.GC.Concurrent"|true &#124; false|Determines whether background garbage collection is enabled (`true`, the default value) or disabled (`false`). For more information, see [Background workstation garbage collection](../../standard/garbage-collection/fundamentals.md#background-workstation-garbage-collection) and [Background server garbage collection](../../standard/garbage-collection/fundamentals.md#background-server-garbage-collection).|
+|"System.GC.RetainVM"|true &#124; false|Determines whether segments that should be deleted are put on a standby list for future use (`true`) or are released back to the operating system (`false`, the default value).|
+|"System.GC.Server"|true &#124; false|Determines whether the application uses server garbage collection (`true`) or workstation garbage collection (`false`, the default value). For more information, see [Configuring garbage collection](../../standard/garbage-collection/fundamentals.md#configuring-garbage-collection).|
```

Comment:
are these supposed to be an exhaustive list of what's supported in runtimeconfig.json in 3.0?

Reply:
I believe @rpetrusha was trying to document all of them.

Reply:
I see, then if you do a search for "System.GC" in eeconfig.cpp, everything you see there should be documented.